pkg/util/oc/pod_commands.go

```diff
@@ -26,7 +26,8 @@ func Exec(t test.TestHelper, podLocator PodLocatorFunc, container string, cmd st
 		checks...)
 }
 
-func MergePatch(t test.TestHelper, ns string, rs string, ptype string, patch string, checks ...assert.CheckFunc) {
+// TODO: avoidError is a hack to avoid error when patching a resource that doesn't exist
+func Patch(t test.TestHelper, ns string, rs string, ptype string, patch string, checks ...assert.CheckFunc) {
```

Comment:
I see this line 30 has been updated in maistra-2.4 branch.
have you fetched and synced the maistra-2.4 branch change ?
https://github.com/maistra/maistra-test-tool/blob/maistra-2.4/pkg/util/oc/pod_commands.go#L30 

It's strange and not clear why this diff is still in your PR.

Reply:
Please check again, I fix some merge conflicts